ext/mbstring/mbstring.c

```diff
@@ -2953,6 +2953,57 @@ PHP_FUNCTION(mb_strtolower)
 	RETURN_STR(mbstring_convert_case(PHP_UNICODE_CASE_LOWER, ZSTR_VAL(str), ZSTR_LEN(str), enc));
 }
 
+static zend_string* php_mb_ulcfirst(zend_string *str, php_case_mode mode, const mbfl_encoding *enc)
+{
+	zend_string *first, *second, *head;
+	first = mb_get_substr(str, 0, 1, enc);
```

Comment:
Codestyle nit: since PHP 8.0 we use C99, which allows writing the declaration and assignment in a single line. I would prefer that.

Reply:
I agree. C99 style avoids unnecessarily using more vertical space on the screen (when someone is reading the code).

Reply:
Thanks! It has changed.